src/app/containers/ATIAnalytics/params/frontpage/buildParams.test.js

```diff
@@ -0,0 +1,92 @@
+import { buildFrontPageATIParams, buildFrontPageATIUrl } from './buildParams';
+import {
+  getPublishedDatetime,
+  getCurrentTime,
+} from '../../../../lib/analyticsUtils';
+
+jest.mock('../../../../lib/analyticsUtils', () => {
+  const utils = jest.requireActual('../../../../lib/analyticsUtils');
+
+  return {
+    ...utils,
+    getPublishedDatetime: jest.fn(),
+    getCurrentTime: jest.fn(),
+  };
+});
+
+getCurrentTime.mockImplementation(() => '00-00-00');
+getPublishedDatetime.mockImplementation(() => '1970-01-01T00:00:00.000Z');
```

Comment:
All this https://github.com/bbc/simorgh/pull/3559/files#diff-969e14df75b9da2e46ff6540eabc15a0R2-R18
Can just be:

```js
import * as analyticsUtils from '../../../../lib/analyticsUtils';

analyticsUtils.getCurrentTime = jest.fn().mockReturnValue('00-00-00');
analyticsUtils.getPublishedDatetime = jest.fn().mockReturnValue('1970-01-01T00:00:00.000Z');
```

Reply:
Fixed!